CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - Added support for creating vectorized UDTFs with `process` method.
 
+### Bug Fixes
+
+- Fixed a bug in local testing file read option `field_optionally_enclosed_by`.
```

Comment:
```suggestion
- Fixed a bug in local testing implementation of DataFrameReader.csv when the optional parameter `field_optionally_enclosed_by` is specified.
```

Reply:
Let's be as specific as possible when we update changelog, this could help users check for potential regression/behavior change.